src/pure.js

```diff
@@ -4,63 +4,90 @@ import {
   getQueriesForElement,
   prettyDOM,
   configure as configureDTL,
+  waitFor as waitForDTL,
+  waitForElementToBeRemoved as waitForElementToBeRemovedDTL,
 } from '@testing-library/dom'
-import act, {asyncAct} from './act-compat'
+import act from './act-compat'
 import {fireEvent} from './fire-event'
 
 configureDTL({
-  asyncWrapper: async cb => {
-    let result
-    await asyncAct(async () => {
-      result = await cb()
-    })
-    return result
-  },
```

Comment:
So to be clear, there's no longer the concept of an `async` act anymore?

Reply:
> So to be clear, there's no longer the concept of an async act anymore?

Nothing changed in that regard. 

I needed to remove to wrapper because  it was used to wrap all the polling in `waitFor`:

```js
await ReactDOMTestUtils.act(async () => {
  await new Promise((resolve) => {
    setInterval(async () => {
      if (
        // That's the condition we wait for
        document.querySelector("main").getAttribute("aria-busy") !== "true"
      ) {
        resolve();
      }
    }, 50);
  });
});
```

and this meant that the screen isn't actually updated during the polling i.e. it'll always reject and then the screen gets updated. This may be fine since we could just check one more time after the timeout but then `interval` becomes useless and the function runtime is always equal to `timeout`.